experimental/packages/opentelemetry-sdk-node/src/sdk.ts

```diff
@@ -139,8 +138,6 @@ export class NodeSDK {
     this._configuration = configuration;
 
     this._resource = configuration.resource ?? new Resource({});
```

Comment:
Hmm, I think this means that the resource will always be set to empty, even if the user does not explicitly provide anything. I think here we have to use the default (just like in the SDKs). 🙂 
```suggestion
    this._resource = configuration.resource ?? Resource.default();
```

Reply:
good catch, I'm changing it in the next commit